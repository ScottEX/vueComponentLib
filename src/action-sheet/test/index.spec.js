import { mount } from '@vue/test-utils';
import ComActionSheetA from '../actionSheet01';
import ComActionSheetB from '../actionSheet02';

test('render demo color', () => {
  const wrapper = mount(ComActionSheetA);
  expect(wrapper).toMatchSnapshot();
});

test('render demo color', () => {
  const wrapper = mount(ComActionSheetB);
  expect(wrapper).toMatchSnapshot();
});
