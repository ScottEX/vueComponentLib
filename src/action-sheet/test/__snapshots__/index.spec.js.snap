// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render demo color 1`] = `
<div class="bob-action-sheet">
  <div class="weui-mask weui-mask_transparent" style="display: none;" name="vux-actionsheet-mask"></div>
  <div class="weui-actionsheet">
    <div class="weui-actionsheet__menu">
      <!---->
    </div>
    <!---->
  </div>
</div>
`;

exports[`render demo color 2`] = `
<div class="bob-action-sheet">
  <div class="weui-mask weui-mask_transparent" style="display: none;" name="vux-actionsheet-mask"></div>
  <div class="weui-actionsheet">
    <div class="weui-actionsheet__menu">
      <div class="weui-actionsheet__title">
        选择拨打电话
      </div>
      <!---->
    </div>
  </div>
  <div class="weui-actionsheet__cancel">
    <!---->
  </div>
</div>
`;
